elasticsearch.go

```diff
@@ -4,7 +4,7 @@ import (
 	"encoding/json"
 	"github.com/buger/elastigo/api"
 	"github.com/buger/elastigo/core"
-	"github.com/buger/gor/proto"
```

Comment:
There is a problem with the latest gor (though that may just be 14.04 not having the correct libpcap version) that results in occasional crashes, so I think we need to base this fork off an older version at least for now.